lib/elasticity/search.rb

```diff
@@ -47,6 +47,7 @@ class Facade
       def initialize(client, search_definition)
         @client            = client
         @search_definition = search_definition
+        @search_definition.body.deep_symbolize_keys!
```

Comment:
Instead of doing this here, please do it inside the constructor of `Search::Definition`: https://github.com/doximity/es-elasticity/blob/fix_hash_strings/lib/elasticity/search.rb#L16